mmc-google-analytics/src/main/java/info/novatec/metricscollector/google/collector/AqeHomePage.java

```diff
@@ -38,9 +44,11 @@ public void collect() {
     private GetReportsResponse requestMetrics() {
         return requestBuilder.prepareRequest()
             .addDimensions(properties.getSharedDimensions())
+            .addDimensions(properties.getAqeHomepage().getSpecificDimensions())
             .addMetrics(properties.getSharedMetrics())
-            //.addDimensionFilters(GA_PAGEPATH, NOT, properties.AqeHomePage().getExcludedUrls())
-            .buildRequest()
+            .addMetrics(properties.getAqeHomepage().getSpecificMetrics())
+            .addDimensionFilters(GA_PAGEPATH, EXACT, Collections.singletonList(AQE_HOME_PAGE_PATH))
+            .buildRequest(properties.getAqeHomepage().getViewId())
             .sendRequest();
```

Comment:
Can you please sort invocations?
.addDimension..
.addDimension..
.addDimension..
.addMetrics..
.addMetrics